huaweicloud/services/elb/data_source_huaweicloud_elb_flavors.go

```diff
@@ -87,11 +90,11 @@ func DataSourceElbFlavorsV3() *schema.Resource {
 	}
 }
 
-func dataSourceElbFlavorsV3Read(d *schema.ResourceData, meta interface{}) error {
+func dataSourceElbFlavorsV3Read(_ context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	config := meta.(*config.Config)
```

Comment:
`config` is same with the package name, it is recommended to rename it

Reply:
done